java/rcloadenv/src/main/java/com/google/cloud/rcloadenv/RuntimeConfigLoaderUtil.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.rcloadenv;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+
+import java.io.IOException;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class RuntimeConfigLoaderUtil {
+
+  private static final String RUNTIMECONFIG_API_ROOT =
+      "https://runtimeconfig.googleapis.com/v1beta1/";
+  private static final String RUNTIMECONFIG_SCOPE =
+      "https://www.googleapis.com/auth/cloudruntimeconfig";
+  private static final String VARIABLES_PATH =
+      "%sprojects/%s/configs/%s/variables?returnValues=true";
+
+  private static final ObjectMapper mapper = new ObjectMapper();
+
+  static Map<String, Object> loadConfig(String projectId, String appName) throws IOException {
```

Comment:
nit: I'd prefer `configName` over `appName` since I think it will be likely in production to have multiple configs to canary new versions.